draft-packages/table/KaizenDraft/Table/Table.tsx

```diff
@@ -350,6 +352,7 @@ export const TableCard = ({
       href={href}
       className={className}
       onClick={onClick as AnchorClickEvent}
+      role="button"
```

Comment:
What's the thinking on this change @kaydbwithers?

Reply:
Thanks @dougmacknz, that slipped in as a mistake, it's been removed now.